### ISSUE TITLE
Product variation attributes

### ISSUE THREAD

**Issue by rotexhawk** [wp-graphql/wp-graphql-woocommerce#130]
I am seeing two two bugs with product attribute variations. The name field returns the wordpress field name. Maybe add `Label` Field as well? Also for sizes the value is empty. 

With the current design I don't know if it's possible to build the default product page. See screenshot. 

![image](https://user-images.githubusercontent.com/12015105/63624933-12beb880-c5cc-11e9-9e85-9afc06a9be8e.png)

![image](https://user-images.githubusercontent.com/12015105/63625257-14d54700-c5cd-11e9-80ba-0724d79582d7.png)

**Comment by rotexhawk** [comment]
Maybe we can leave the current data as is and have an endpoint to get the list of all product attributes?

**Comment by rotexhawk** [comment]
Yeah, that might be a bit tricky. Honestly for version one, it will be easier to just return the values as they are `entered/stored`.

**Comment by kidunot89** [comment]
@rotexhawk I think I stumbled on to the solution for this issue. The attributes and there options are stored on the parent product as well. If you run a cross-examination for any attributes on the variation with `""` for the value, you can use the render a `<select />` option or whatever using the data from the corresponding `attribute` or the parent Product

**Comment by kidunot89** [comment]
@rotexhawk Is this still an issue? I've long since lost track of reasoning for this issue.

**Comment by HiepVu511** [comment]
I think we still need a `label` field. Right now there is no easy way to dynamically query that data

**Comment by thetre97** [comment]
Linked with #280

**Comment by Jommartinez** [comment]
I think we still need to be able to export the custom attributes of product variations. Has anyone solved it?

**Comment by jviana-cx** [comment]
+1 Still need this
